a10_octavia/controller/worker/controller_worker.py

```diff
@@ -236,7 +237,7 @@ def update_health_monitor(self, health_monitor_id, health_monitor_updates):
 
     def create_listener(self, listener_id):
         """Function to create listener for A10 provider"""
-
+        time.sleep(60)
```

Comment:
Don't use sleep(60) but use a general API to check/re-check and wait instead.

Reply:
Fixed